### PR TITLE
feat: IOS GameLoop addition

### PR DESCRIPTION
**Pull request by axelzuziak-gogo**
This PR adds iOS example project with a game loop.

Scripts for generating IPA file from iOS source code will be added in:
https://github.com/Flank/flank/issues/1345

Support for game loop in Flank will be fully implemented in:
https://github.com/Flank/flank/issues/1341

.

## Checklist

- [x] Documented
- [x] Unit tested

_____

Bottom note: Flank doesn't support gameloop mode for iOS yet. 
Flank requires xctestrun-file which doesn't exist in gameloop mode. #1341

### CODE REVIEW

**Comment by axelzuziak-gogo** [comment]
@Sloox Added temporary bash scripts:
* export.sh - will generate IPA
* ftl.s - will run gcloud command
You can verify the test project using them.

File: test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-gameloop-ios.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+gcloud:
+  app: test_projects/ios/FlankGameLoopExample/ipa/FlankGameLoopExample.ipa
+  type: game-loop
+  scenario-numbers:
+      - 1
+  device:
+      - model: iphone11
+        version: 13.6
+        orientation: portrait
+        locale: en_US
```
Comment (adamfilipow92) [inline test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-gameloop-ios.yml:10]: - When I try to run this test from Debug.kt Flank returns

```
Both of following options must be specified [test, xctestrun-file].

Process finished with exit code 2

```

- Blank line missing

Comment (axelzuziak-gogo) [reply test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-gameloop-ios.yml:10]: @adamfilipow92 FTL with gameloop mode doesn't need xctestrun-file,
however Flank currently requires it. @Sloox will fix this in next a PR

Comment (adamfilipow92) [reply test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-gameloop-ios.yml:10]: Thanks! I try to run in on gcloud cli!

**Review by adamfilipow92 — APPROVED** [review]
Tested on gcloud cli
```
gcloud alpha firebase test ios run \
 --type game-loop --app FlankGameLoopExample.ipa --scenario-numbers 1 \
 --device model=iphone11,version=13.6

```

Test pass

```

┌─────────┬───────────────────────────┬──────────────┐
│ OUTCOME │      TEST_AXIS_VALUE      │ TEST_DETAILS │
├─────────┼───────────────────────────┼──────────────┤
│ Passed  │ iphone11-13.6-en-portrait │ --           │
└─────────┴───────────────────────────┴──────────────┘

```

File: test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-gameloop-ios.yml (hunk 1 above)
Comment (Sloox) [reply test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-gameloop-ios.yml:10]: @adamfilipow92 this wont work until @jan-gogo applies a fix that will come in #1341 . However #1341 is blocked until other tickets are closed (including this one.)

**Review by pawelpasterz — APPROVED** [review]

**Pull request merged**